protocol/communities/community_event.go

```diff
@@ -246,23 +237,19 @@ func (o *Community) updateCommunityDescriptionByCommunityEvent(communityEvent Co
 		o.config.CommunityDescription.Tags = communityEvent.CommunityConfig.Tags
 
 	case protobuf.CommunityEvent_COMMUNITY_MEMBER_TOKEN_PERMISSION_CHANGE:
-		_, exists := o.config.CommunityDescription.TokenPermissions[communityEvent.TokenPermission.Id]
-		if exists {
-			_, err := o.updateTokenPermission(communityEvent.TokenPermission)
-			if err != nil {
-				return err
-			}
-		} else {
-			_, err := o.addTokenPermission(communityEvent.TokenPermission)
+		if o.IsControlNode() {
```

Comment:
```suggestion
		if o.IsPrivilegedMember(o.MemberIdentity() {
```
Other admins should see pending changes.